bin/build-release-ccd-definition.sh

```diff
@@ -9,17 +9,17 @@ if [[ ${environment} != "prod" && ${environment} != "aat" && ${environment} != "
   exit 1
 fi
 
+if [ ${environment} == "prod" ]; then
```

Comment:
Can set `excludedFilenamePatterns` to `"-e UserProfile.json"` by default and then just have the if append if the environment is prod. (`excludedFilenamePatterns="${excludedFilenamePatterns},*-nonprod.json"`)